### ISSUE TITLE
Add swap metric

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-telemetry-streaming#120]
### Is your feature request related to a problem? Please describe.
system swap is a metric which is important for operations. I often see it monitored on legacy systems (snmp poll).
That metric is not provided at this time.

### Describe the solution you'd like
Include the following in the properties.json
```
        "swap": {
            "structure": { "parentKey": "system" },
            "key": "/mgmt/tm/sys/memory::sys/memory/memory-host",
            "normalization": [
                {
                    "runFunctions": [
                        { "name": "getPercentFromKeys", "args": { "totalKey": "swapTotal", "partialKey": "swapUsed", "nestedObjects": true } }
                    ]
                }
            ]
        },
```

### Describe alternatives you've considered
none

**Comment by dstokesf5** [comment]
Added to our internal product backlog as AUTOTOOL-2314. Thank you for getting this work started for us.

**Comment by dstokesf5** [comment]
This issue was resolved in the 1.19.0 release.
